spark-connector/spark-common/src/main/java/org/apache/gravitino/spark/connector/utils/ConnectorUtil.java

```diff
@@ -37,4 +41,11 @@ public static String removeDuplicateSparkExtensions(
         .reduce((element1, element2) -> element1 + ConnectorConstants.COMMA + element2)
         .orElse("");
   }
+
+  @VisibleForTesting
```

Comment:
@VisibleForTesting ?

Reply:
any problem here?

Reply:
it's used in different packages, should remove `VisibleForTesting`

Reply:
removed.